app/components/code-mirror.js

```diff
@@ -0,0 +1,188 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+import {
+  EditorView,
+  crosshairCursor,
+  drawSelection,
+  dropCursor,
+  highlightActiveLine,
+  highlightActiveLineGutter,
+  highlightSpecialChars,
+  highlightTrailingWhitespace,
+  highlightWhitespace,
+  keymap,
+  lineNumbers,
+  placeholder,
+  rectangularSelection,
+  scrollPastEnd,
+} from '@codemirror/view';
+import { Compartment, EditorState } from '@codemirror/state';
+import { defaultKeymap, history, historyKeymap, indentWithTab } from '@codemirror/commands';
+import { highlightSelectionMatches } from '@codemirror/search';
+import { autocompletion, closeBrackets, closeBracketsKeymap, completionKeymap } from '@codemirror/autocomplete';
+import { unifiedMergeView } from '@codemirror/merge';
+import {
+  LanguageDescription,
+  bracketMatching,
+  defaultHighlightStyle,
+  foldGutter,
+  foldKeymap,
+  indentOnInput,
+  indentUnit,
+  syntaxHighlighting,
+} from '@codemirror/language';
+import { languages } from '@codemirror/language-data';
+import { markdown } from '@codemirror/lang-markdown';
+
+import * as THEME_EXTENSIONS from '@uiw/codemirror-themes-all';
```

Comment:
@VasylMarchuk I'm not sure how this works with bundling / tree-shaking - it looks like they have separate packages for each theme, maybe we can just load the ones we want? That'd be `github-dark` and `github-dark`

Reply:
Ah, just saw your note in the description now re: this (hadn't seen that before)

Reply:
Done, only kept GitHub Light & Dark!